### ISSUE TITLE
pad sequence in my dataset

### ISSUE THREAD

**Issue by geekgitcode** [bentrevett/pytorch-sentiment-analysis#107]
hello~ Thank you for your outstanding work!
i got some problems when i wanted to use my own custom dataset.

when i have train/validation dataset and got my model, how can use the model in the test dataset? my test dataset has no labels and only has `['ID', 'Phrase']`, i made my dataset and data_loader as this:
`test_dataset = data.TabularDataset(
    path='./test.csv', format='csv', skip_header=True, fields=[("PhraseId", None), ("Phrase", TEXT)] 
)`
`test_loader = data.Iterator(dataset=test_dataset, 
                            batch_size=batch_size, 
                            train=False,
                            sort=False,
                            sort_within_batch=False,
                            device=device)`
because i don't want to break the order of my test dataset so i use `data.Iterator` and the final answer seemed to get some problems. how can i solve this problem in the situation of protecting the order of my test dataset?

this is my lstm using the pack skill
```
class RNN(nn.Module):
    def __init__(self, vocab_size, embedding_dim, hidden_dim, num_layers, num_classes, pad_idx):
        super(RNN, self).__init__()
        self.embedding = nn.Embedding(vocab_size, embedding_dim, padding_idx=pad_idx)
        self.hidden_dim = hidden_dim
        self.num_layers = num_layers
        self.lstm = nn.LSTM(embedding_dim, hidden_dim, num_layers)
        self.fc = nn.Linear(hidden_dim, num_classes)
    
    def forward(self, x, x_lengths):
        # x = [seq len, batch size]
        embedded = self.embedding(x)   # embedded = [seq len, batch size, emb dim]
        
        # pack sequence lengths need to be on CPU! 
        packed_embedded = nn.utils.rnn.pack_padded_sequence(embedded, x_lengths.to('cpu'), enforce_sorted=False)
        
        # Set initial hidden and cell states 
        # h0 = torch.zeros(self.num_layers, embedded.size(1), self.hidden_dim).to(device) 
        # c0 = torch.zeros(self.num_layers, embedded.size(1), self.hidden_dim).to(device)
        
        # Forward propagate LSTM
        # packed_out, _ = self.lstm(packed_embedded, (h0, c0))  # packed_out: tensor of shape (seq_length, batch_size, hidden_dim)
        packed_out, _ = self.lstm(packed_embedded)
        
        #unpack sequence
        out, out_lengths = nn.utils.rnn.pad_packed_sequence(packed_out)
        
        # Decode the hidden state of the last time step
        out = self.fc(out[-1, :, :])    # out (seq_length, batch_size, hidden_dim) ->  (batch_size, n_classes)
        return out
```
